agent-server/bin/create-mcp.sh

```
@@ -0,0 +1,148 @@
+DESCRIPTION=${DESCRIPTION:-"An MCP server for ${SERVER_NAME}"}
+
+# Check if server already exists
+if [ -d "$SERVERS_DIR/$SERVER_NAME" ]; then
+  log_warning "Warning: Server '$SERVER_NAME' already exists. Overwrite? (y/N)"
+  read -p "" overwrite
+  if [[ ! "$overwrite" =~ ^[Yy]$ ]]; then
+    log_info "Operation cancelled."
+    exit 0
+  fi
+  # Remove existing directory
+  rm -rf "$SERVERS_DIR/$SERVER_NAME"
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)